css/prism.css

```diff
@@ -36,6 +36,29 @@
 }
 
 /* Token styles */
+
+code[class*="language-"],
+pre[class*="language-"] {
+	background: #27212e;
+	color: #ffffff;
+	font-family: Consolas, Monaco, "Andale Mono", "Ubuntu Mono", monospace; /* this is the default */
+	/* The following properties are standard, please leave them as they are */
+	direction: ltr;
+	text-align: left;
+	white-space: pre;
+	word-spacing: normal;
+	word-break: normal;
+	line-height: 1.5;
+	-moz-tab-size: 2;
+	-o-tab-size: 2;
+	tab-size: 2;
+	/* The following properties are also standard */
+	-webkit-hyphens: none;
+	-moz-hyphens: none;
+	-ms-hyphens: none;
+	hyphens: none;
+}
+
```

Comment:
Are we changing these values from default? If they're defined elsewhere, then we can omit them. Have you tried without them?

Reply:
Apparently there is no default so this is adding values.